apps/content-analysis-webworker/webpack.config.js

```diff
@@ -0,0 +1,23 @@
+const webpack = require( "webpack" );
+const HtmlWebpackPlugin = require( "html-webpack-plugin" );
+const path = require( "path" );
+
+module.exports = {
+	plugins: [
+		new webpack.ProvidePlugin({
+			process: 'process/browser',
+		}),
+		new HtmlWebpackPlugin({
+			template: path.join(__dirname, 'src/index.html'),
+			filename: 'index.html',
+		}),
+	],
+	devServer: {
+		static: {
+			directory: path.join(__dirname, 'dist'),
+		},
+		compress: true,
+		port: 9000,
```

Comment:
Somehow, in my computer, port `9000` is also occupied by my PHPStorm. So I cannot start the app without closing PHPStorm, was this intentional?

Reply:
No, let's pick another port! 👍